File: week-02/day-03/is_in_list.py
# Check if list contains all of the following elements: 4,8,12,16
# Create a function that accepts list_of_numbers as an input
# it should return "True" if it contains all, otherwise "False"

list_of_numbers = [2, 4, 6, 8, 10, 12, 14, 16]
x = [4,8,12,16,33]

def listnums(y):

    for i in x:
        if i in y:
            print(True)
        else:
            print(False)

listnums(list_of_numbers)

File: week-07/day-02/is-in-list.js
'use strict';
// Check if array contains all of the following elements: 4,8,12,16
// Create a 'numChecker' function that accepts 'listOfNumbers' as an input
// it should return "true" if it contains all, otherwise "false"

var listOfNumbers = [2, 4, 6, 8, 10, 12, 14, 16];
var test_array = [ 4, 8, 12, 16 ];
var control_array =[];
function numChecker(array){
  for(var i = 0; i < array.length; i++){
    if(listOfNumbers.indexOf(array[i]) != -1){
      control_array.push(array[i])
    }
  }
  if(control_array.length == test_array.length){
    return true
  }else{
    return false
  }
}

console.log(numChecker(test_array))
